docs/source/models/risk_effects/child_growth_failure/cgf_2021/index.rst

```diff
@@ -98,10 +98,20 @@ and cause metric affected (e.g., incidence), and the simulant's age (e.g., 6-11
 Putting this together, a single relative risk might be the RR for **WAZ** on **malaria incidence** 
 for a simulant in the **moderate** exposure category who is **6-11 months** old. 
 
-Relative risk values can be pulled using the following code::
+For stunting and underweight, relative risk values can be pulled using the following code::
 
   rrs = get_draws(gbd_round_id=7, year_id=2021, gbd_id_type='rei_id', gbd_id=[241,240,94], source='rr', decomp_step='iterative')
 
+Wasting relative risks will be generated separately to accomodate the sub-exposures in the 
+MAM (cat2) category in our :ref:`wasting risk exposure model <2021_risk_exposure_wasting_state_exposure>`. 
+Until these relative risk estimates are available, assume wasting relative risks consistent
+with those pulled from GBD with both MAM sub exposures ("Better" MAM/cat2.5 and "Worse" MAM/cat2.0) 
+having the same relative risk values as the overall MAM category cat2.
```

Comment:
Making them different with a placeholder RR (e.g. 2.0) could make testing easier.

Reply:
Doing something this simple would break our PAFs and we would no longer validate, so I didn't want to take this approach. We could decompose the RRs and make them a simple function so that they aggregate back up to the RR of the overall MAM category, but I thought it might be too much annoying coding for Hussain that will be thrown out when we will hopefully have the real RRs soon - do you think it would be worth it?